developer-docs/bolt-server.md

```diff
@@ -223,6 +226,19 @@ curl -v --cacert $BOLT_CACERT --cert $BOLT_CERT --key $BOLT_KEY $BOLT_ROOT
 
 ## Making requests
 
+### With the ruby client
+
+There is a simple ruby client that can be used to make requests to a local
+bolt_server during development at `scripts/server_client.rb`. This server
+expects to use the puppet-server container and target nodes from bolts spec
+environment so run `docker-compose up -d --build` from the spec directory before
+trying to use it.
+
+```
+bundle exec scripts/server_client.rb sample::echo ssh '{"message": "hey"}'
```

Comment:
I get the following trying to run this:
```
OpenSSL::SSL::SSLError - SSL_connect returned=1 errno=0 state=error: certificate verify failed (unspecified certificate verification error):
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/protocol.rb:44:in `connect_nonblock'
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/protocol.rb:44:in `ssl_socket_connect'
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/http.rb:981:in `connect'
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/http.rb:920:in `do_start'
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/http.rb:909:in `start'
boltserver_1  | 	/usr/local/lib/ruby/2.5.0/net/http.rb:1455:in `request'
boltserver_1  | 	/bolt-server/lib/bolt_server/file_cache.rb:46:in `request_file'
boltserver_1  | 	/bolt-server/lib/bolt_server/file_cache.rb:72:in `download_file'
boltserver_1  | 	/bolt-server/lib/bolt_server/file_cache.rb:95:in `block in update_file'
```

Reply:
Do you have an existing setup for bolt-server somewhere?

Reply:
did you rebuild the spec compose after my pre-lunch push? I had to regenerate the certs to include the alt name spec_puppetserver_1

Reply:
Yeah, I'm pointing to the ssl files in bolt/spec/fixtures...
```
bolt-server: {
  ssl-cert: "/home/lucy/githubs/bolt/spec/fixtures/ssl/cert.pem"
  ssl-key: "/home/lucy/githubs/bolt/spec/fixtures/ssl/key.pem"
  ssl-ca-cert: "/home/lucy/githubs/bolt/spec/fixtures/ssl/ca.pem"
}
```
In ~/.puppetlabs/bolt-server.conf

Reply:
Ooo, no I did not